File: testprojects/rel_scala_2_4_project/pipelines/Pipeline_Scala2-11_Spark2-4/code/src/main/scala/io/prophecy/pipelines/pipeline_scala211_spark24/graph/Script_1.scala
package io.prophecy.pipelines.pipeline_scala211_spark24.graph

import io.prophecy.libs._
import io.prophecy.pipelines.pipeline_scala211_spark24.config.ConfigStore._
import io.prophecy.pipelines.pipeline_scala211_spark24.config.Context
import io.prophecy.pipelines.pipeline_scala211_spark24.udfs.UDFs._
import io.prophecy.pipelines.pipeline_scala211_spark24.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Script_1 {
  def apply(context: Context, in0: DataFrame): DataFrame = {
    val spark = context.spark
    val Config = context.config
    print("Successfully Executed.")
    var out0=in0
    out0
  }

}

File: testprojects/master_python_mainone/pipelines/PYTHON_DEP_MANAGEMENT_ALL/code/job/graph/Script_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def Script_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    out0 = in0

    return out0
